libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -108,21 +103,56 @@ Json::Value const& ASTJsonConverter::json()
 	return m_astJson;
 }
 
+bool ASTJsonConverter::visit(SourceUnit const&)
+{
+	Json::Value children(Json::arrayValue);
+
+	m_astJson["name"] = "root";
```

Comment:
Ok, then please just rename it. I think we called it `root` because we did not have a better name, now we have.